### PR TITLE
nixos/test-driver: unrevert logging changes

### PR DESCRIPTION
**Pull request by JJJollyjim**
###### Motivation for this change

Hi all

I tried to clean up the mess from #96699 :(

@ofborg test initrd-network-ssh ipv6

cc @Mic92 @andersk @worldofpeace 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/lib/testing-python.nix [hunk 1]
```diff
@@ -65,6 +65,12 @@ rec {
           mkdir -p $out
 
           LOGFILE=/dev/null tests='exec(os.environ["testScript"])' ${driver}/bin/nixos-test-driver
+
+          for i in */xchg/coverage-data; do
+            mkdir -p $out/coverage-data
+            mv $i $out/coverage-data/$(dirname $(dirname $i))
+          done
+>>>>>>> parent of 59b6664f15c... Revert "Merge pull request #96254 from Mic92/logging"
```
Comment (Mic92) [inline nixos/lib/testing-python.nix:73]: This should not be here:

**Review by worldofpeace — CHANGES_REQUESTED** [review]
see above

**Comment by SuperSandro2000** [comment]
@JJJollyjim ping

**Comment by JJJollyjim** [comment]
@SuperSandro2000 I have given up on contributing to nixpkgs, sorry

**Comment by SuperSandro2000** [comment]
@mic92 Do you think this is still relevant?

**Comment by Mic92** [comment]
It is.

**Pull request closed without merging**